benchmarks/perfetto/s3_utils.py

```diff
@@ -0,0 +1,35 @@
+import os
+from pathlib import Path
+from . import BENCHMARK_ROOT
+
+
+def decompress_s3_data(s3_tarball_path: Path):
+    assert str(s3_tarball_path.absolute()).endswith(
+        ".tar.gz"
+    ), f"Expected .tar.gz file path but get {s3_tarball_path}."
+    import tarfile
+
+    # Hide decompressed file in .data directory so that they won't be checked in
+    decompress_dir = os.path.join(BENCHMARK_ROOT, ".data", s3_tarball_path.name.rstrip(".tar.gz"))
```

Comment:
Do you want to extract `.data` into a constant?

Reply:
We should use `BENCHMARK_DATA_DIR` here.